src/content/core/credentials/environment-variables.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: Environment Variables
+description: Specify your credentials using environment variables
+sidebar_label: Environment Variables
+id: environment-variables
+---
+
+Environment Variables provide another way to specify your credentials. This is the recommended method when scripting tasks or when [creating Preview Environments](previews/index.mdx).
+
+The supported environment variables are:
+
+- `OKTETO_TOKEN`: Specifies the [Personal Access Token](core/credentials/personal-access-tokens.mdx) or [Admin Access Token](/docs/admin/dashboard/#admin-access-tokens) to use (Required)
+- `OKTETO_CONTEXT`: Specifies the Okteto URL. If missing, it defaults to https://okteto.example.com (Required)
+- `OKTETO_NAMESPACE`: Specifies the Namespace to use. If missing, it defaults to the Namespace of the current context (Optional)
+
+:::tip
+
+You can use the `.env` file to store your credentials. See [Okteto Variables](/docs/core/okteto-variables) to learn more.
```

Comment:
I don't remember the implications, but we link to the `mdx` file. This should be `(core/okteto-variables.mdx)`